doc/rest-api.md

```diff
@@ -2273,7 +2273,7 @@ Input (none at present):
  * Description: create a new storage volume on a given storage pool
  * Introduced: with API extension `storage`
  * Authentication: trusted
- * Operation: sync
+ * Operation: sync/async
```

Comment:
Should be:

    Operation: sync or async (when copying)

To clarify.

Reply:
added